values-schema.yaml

```diff
@@ -377,6 +411,10 @@ definitions:
       id:
         $ref: '#/definitions/idName'
         description: Must be the same as the name.
+      jobs:
```

Comment:
Right now  '#/definitions/jobs` is already an array. But If you make 
a definition for single job the this statement should be fine.

Reply:
Yes that makes sense. I think having a job definition with a single object and define arrays for multiple jobs elsewhere is even better, right?